package.json

```diff
@@ -34,5 +34,14 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/slackhq/tree-sitter-hack.git"
-  }
+  },
+  "tree-sitter": [
+    {
+      "scope": "source.hack",
+      "file-types": [
+        "hack"
+      ],
+      "first-line-regex": "#!.*\\bhack$"
```

Comment:
This might not be the first-line-regex we want. Unfortunately, however, it looks like tree-sitter only supports one first-line-regex since this field doesn't allow arrays. This means we have to decide between supporting `#!/usr/bin/env hhvm` or `<?hh`. Since the shebang line is optional, I'm more in favor of changing this to check for `<?hh` over correcting it to `hhvm`.


See https://docs.hhvm.com/hack/source-code-fundamentals/program-structure#legacy-files

Reply:
We could also just write a fancier regex that checks for either I guess 🙃